Formula/helidon.rb

```diff
@@ -0,0 +1,21 @@
+class Helidon < Formula
+  desc "Command line tool for Helidon application development"
+  homepage "https://helidon.io/"
+  url "https://search.maven.org/remotecontent?filepath=io/helidon/build-tools/cli/helidon-cli-impl/3.0.4/helidon-cli-impl-3.0.4.zip"
```

Comment:
When I try this URL behind the Oracle proxy (using browser or curl) I get a 403. I also saw this: https://central.sonatype.org/faq/what-happened-to-search-maven-org/ 

Have you checked with homebrew why they don't like https://repo1.maven.org/maven2/io/helidon/build-tools/cli/helidon-cli-impl/3.0.4/helidon-cli-impl-3.0.4.jar ?

Reply:
See the following: https://github.com/orgs/Homebrew/discussions/4276.